TDDPOC/TDDPOC.Test/WeatherForecastTest.cs

```diff
@@ -0,0 +1,64 @@
+using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using TDDPOC.Controllers;
+using TDDPOC.Service;
+
+namespace TDDPOC.Test
+{
+    public class WeatherForecastTest
+    {
+        private WeatherForecastController _controller;
```

Comment:
This declarations should have readonly, please read about this

Reply:
updated the declarations to readonly